regtests/client/python/cli/polaris_cli.py

```diff
@@ -55,6 +67,27 @@ def execute(args=None):
                 PolarisCli._try_print_exception(e)
                 sys.exit(1)
 
+    @staticmethod
+    def _merge_with_config(options):
+        config = PolarisCli._load_config()
```

Comment:
Could we do a quick rename of this to file_config or something like that? Just to make it clear the difference between options and config. The lines below are a little complicated to me otherwise